reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+        
+
+  
+schemes:
+  - https
+  
+host: your-store-domain.myshopify.com
+
+paths:
+  /admin/draft_orders.json:
+    get:
+      summary: Retrieve draft orders
+      description: You can use this endpoint to return a list of draft order objects.  This can be useful for a variety of applications including draft order printing.  For `line_items` you can use the value of the `custom` field (true/false) to differentiate between custom line items and product variant line items.
+      parameters:
+      - $ref: '#/parameters/fields'
+      - $ref: '#/parameters/limit'
+      - $ref: '#/parameters/page'
+      - $ref: '#/parameters/since_id'
+      - $ref: '#/parameters/status'
+      - $ref: '#/parameters/ids'
+    
+      tags:
+        - Get
+
+      responses:
+        '200':
+          description: List of DraftOrder
+          schema:
+            $ref: '#/definitions/DraftOrder'
+        
+        '400':
+          description: General API error
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+       
+          
+            
+      x-code-samples:
+        - lang: curl
+          source:  |
+            curl -X GET -H "Authorization: Basic YOUR-API-KEY=" "https://your_store_domain.myshopify.com/admin/draft_orders.json"
+      
+        - lang: Ruby
+          source: |
+              # load the gem
+              require 'swagger_client'
+              
+              api_instance = SwaggerClient::GetApi.new
+              
+              opts = { 
+                fields: "fields_example", # String | Comma-separated list of fields to include in the response
+                limit: 789, # Integer | Page size
+                page: 789 # Integer | Page to show
+              }
+              
+              begin
+                result = api_instance.admin_draft_orders_json_get(opts)
+                p result
+              rescue SwaggerClient::ApiError => e
+                puts "Exception when calling GetApi->admin_draft_orders_json_get: #{e}"
+              end
+            
+            
+  
+    post:
+      summary: Create a draft order
+      description: |
+          
+          Using the Draft Orders API you can create draft orders using product variant line items, or your own custom line items.  This enables you to create one-off products that exist outside of your regular inventory management system.  To create a product variant draft order, simply specify a `variant_id` `quantity` and `discount` (if applicable).  Create a custom line item by supplying `title`, `price`, `taxable` and `quantity`. 
```

Comment:
I would vote to keep it as "Create a draft order" - partly because the code examples will do just that, and the created resource will live in the Draft orders section of the admin.